api/src/main/java/marquez/api/OpenLineageResource.java

```diff
@@ -80,10 +91,48 @@ private int determineStatusCode(Throwable e) {
   @GET
   @Consumes(APPLICATION_JSON)
   @Produces(APPLICATION_JSON)
+  @Path("/lineage")
   public Response getLineage(
       @QueryParam("nodeId") @NotNull NodeId nodeId,
-      @QueryParam("depth") @DefaultValue(DEFAULT_DEPTH) int depth)
-      throws ExecutionException, InterruptedException {
+      @QueryParam("depth") @DefaultValue(DEFAULT_DEPTH) int depth) {
     return Response.ok(lineageService.lineage(nodeId, depth)).build();
   }
+
+  @Timed
+  @ResponseMetered
+  @ExceptionMetered
+  @GET
+  @Path("/events/lineage")
+  @Produces(APPLICATION_JSON)
+  public Response getLineageEvents(
+      @QueryParam("before") @DefaultValue("2030-01-01T00:00:00+00:00") ZonedDateTimeParam before,
+      @QueryParam("after") @DefaultValue("1970-01-01T00:00:00+00:00") ZonedDateTimeParam after,
+      @QueryParam("limit") @DefaultValue("100") @Min(value = 0) int limit) {
+    final List<LineageEvent> events =
+        openLineageDao.getAllLineageEvents(before.get(), after.get(), limit);
+    return Response.ok(new Events(events)).build();
+  }
+
+  @Timed
+  @ResponseMetered
+  @ExceptionMetered
+  @GET
+  @Path("/namespace/{namespace}/events/lineage")
```

Comment:
if it is the same resource, but filtered, shouldn't this look like:
```
/events/lineage?namespace=some-namespace
```

Reply:
@wslulciuc suggested that it would be not consistent with how other endpoint are defined in Marquez, which I agree with: https://github.com/MarquezProject/marquez/pull/2070#discussion_r964417166

Reply:
ok, let's stay consistent